src/common/CommonComponent.js

```diff
@@ -0,0 +1,49 @@
+import { TagNames } from '@/utils';
+
+export class CommonComponent {
+  #element;
+
+  constructor(element) {
+    this.#element = document.createElement(element);
+  }
+
+  getComponent() {
+    return this.#element;
+  }
+
+  addClassName(classname) {
+    this.#element.className = classname;
+    return this.#element;
+  }
+
+  addValue(value) {
+    this.#element.value = value;
+    return this.#element;
+  }
+
+  addInnerHTML(text) {
+    this.#element.innerHTML = text;
+    return this.#element;
+  }
+
+  addAttribute(attribute, value) {
+    this.#element.setAttribute('type', attribute);
+    this.#element.setAttribute('value', value);
+    return this.#element;
+  }
+
+  addOptions(item) {
+    const option = new CommonComponent(TagNames.OPTION);
+    option.addInnerHTML(item.name);
+    option.addValue(item.symbol);
+    this.getComponent().appendChild(option.getComponent());
+  }
+
+  renderElement(toClass, className) {
+    return document.querySelector(toClass).appendChild(className);
```

Comment:
You shouldn't use querySelector for nesting children. You should return elements in classes and use method .appendChild into rendering wrapper.
For example:
`
class InputWrapper {
  render() {
    const inputInstance = new Input().render();
    this.appendChild(inputInstance);
    return this.getComponent();
  }
}
`